packages/amplify-provider-awscloudformation/src/pre-push-cfn-processor/cfn-pre-processor.ts

```diff
@@ -1,7 +1,22 @@
-import { pathManager, readCFNTemplate, writeCFNTemplate } from 'amplify-cli-core';
+import { stateManager,
+  pathManager,
+  readCFNTemplate,
+  writeCFNTemplate,
+  CustomIAMPolicy,
+  CustomIAMPolicySchema,
+  CustomIAMPolicies,
+  customExecutionPolicyForFunction,
+  customExecutionPolicyForContainer,
+  CustomPoliciesFormatError,
+  CustomIAMPoliciesSchema,} from 'amplify-cli-core';
 import * as path from 'path';
 import { ProviderName as providerName } from '../constants';
 import { prePushCfnTemplateModifier } from './pre-push-cfn-modifier';
+import { Fn, Template } from 'cloudform-types';
+import { printer } from 'amplify-prompts';
+import Ajv from "ajv";
+import * as iam from '@aws-cdk/aws-iam';
```

Comment:
We are using cloudform here , we can remove this.

Reply:
I am not quite sure what you mean here